.github/workflows/main.yml

```diff
@@ -87,7 +87,7 @@ jobs:
           BUCKET_NAME: ${{ secrets.BUCKET_NAME }}
           BUCKET_MEDIA_FOLDER: ${{ secrets.BUCKET_MEDIA_FOLDER }}
         run: |
-          PORT=8002 docker-compose up -d --build
```

Comment:
Why change the port?
Also, setting it would "document" that the number "8080"  in e2E test is a port

Reply:
It's more that I remove the port flexibility, since it can be changed in the docker-compose. So I picked a 80xx port out of convenience. Not sure about what you mean by "document" ?